cdv/cmds/rpc.py

```diff
@@ -273,8 +271,11 @@ async def do_command():
         try:
             node_client: FullNodeRpcClient = await get_client()
             if transaction_id:
-                items = {}
-                items[transaction_id] = await node_client.get_mempool_item_by_tx_id(hexstr_to_bytes(transaction_id))
+                items = {
+                    transaction_id: await node_client.get_mempool_item_by_tx_id(
+                        bytes32.from_bytes(hexstr_to_bytes(transaction_id))
```

Comment:
Consistent with the other changes, this should be bytes32.from_hexstr yeah?

Reply:
Passes lint so I'm not sure, I'll leave it as is.

Reply:
Oh wait yep I see what you mean